zero_bin/common/src/build_version.rs

```diff
@@ -0,0 +1,41 @@
+use anyhow::Context as _;
+use vergen::{BuildBuilder, Emitter, RustcBuilder};
+
+fn main() -> anyhow::Result<()> {
```

Comment:
@atanmarko @Nashtare do we need to name this func as main in build scripts or we can name it anything?

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html